frontend/layouts/default.vue

```diff
@@ -127,13 +179,30 @@
         <span>{{ $t('auth.login.label') }}</span>
       </v-tooltip>
     </v-app-bar>
-    <v-main class="min-height-full-display">
+    <v-main>
       <v-container fluid ma-50>
         <v-breadcrumbs :items="$store.state.breadcrumb.items" />
         <nuxt />
       </v-container>
     </v-main>
-    <philo-footer />
```

Comment:
why are you removing the footer? we are losing some options

Reply:
let me review, there's some version difference i think here